tox.ini

```diff
@@ -0,0 +1,93 @@
+[tox]
+envlist =
+    test-{py36,py37,py38,py39}
+    integration-test-{py36,py37,py38,py39}
+    isort
+    flake8
+    black
+    blackdoc
+    mypy
+
+[testenv:test-py36]
+basepython = python3.6
+extras =
+    tests
+    optional
+deps =
+    scikit-optimize
```

Comment:
We don't need `scikit-optimize` except for integration tests.

Reply:
I add scikit-optimize due to the following comment:
https://github.com/optuna/optuna/blob/fbe4e6160fddf811659ad6d34ad7658db1f3ff6c/.github/workflows/tests.yml#L66-L69

Is it no longer needed?

Reply:
I tried to remove scikit-optimize, then unit tests are failed. It seems required for now.

Reply:
Sorry for the confusion. I missed the test case. You are right. Let me resolve this conversation.